src/components/Tour.vue

```diff
@@ -76,6 +110,17 @@ export default {
 					}
 					steps.push('ide-viewer');
 					break;
+				case 'connect':
+					steps.push('connect-url');
+					steps.push('connect-history');
+					break;
+				case 'login':
+					steps.push(['login-options', 'left']);
```

Comment:
Do you actually need to provide the placement options here? I think we should only provide them if the default doesn't work well.

Reply:
Yes, I included the placement options on purpose because I don't want the help bubbles to hide the form, but prefer them to open over the space that is blank anyway.

Reply:
Okay, fine with me.